### PR TITLE
store: ignore the store if the capacity is 0B

### LINKED ISSUE #6252: metrics: low space maybe wrong if tikv panic
### What did you expect to see?
image

We should ignore the store if the capacity is 0B.
```
{
  "count": 4,
  "stores": [
    {
      "store": {
        "id": 4,
        "address": "db-tikv-0.db-tikv-peer.tidb1379661944632844070.svc:20160",
        "labels": [
          {
            "key": "host",
            "value": "ip-172-30-174-121.ap-northeast-1.compute.internal"
          },
          {
            "key": "region",
            "value": "ap-northeast-1"
          },
          {
            "key": "zone",
            "value": "ap-northeast-1d"
          }
        ],
        "version": "6.5.0",
        "peer_address": "db-tikv-0.db-tikv-peer.tidb1379661944632844070.svc:20160",
        "status_address": "db-tikv-0.db-tikv-peer.tidb1379661944632844070.svc:20180",
        "git_hash": "47b81680f75adc4b7200480cea5dbe46ae07c4b5",
        "start_timestamp": 1675135145,
        "deploy_path": "/",
        "last_heartbeat": 1679898149244798978,
        "state_name": "Offline"
      },
      "status": {
        "capacity": "0B",
        "available": "0B",
        "used_size": "0B",
        "leader_count": 0,
        "leader_weight": 1,
        "leader_score": 0,
        "leader_size": 0,
        "region_count": 3345,
        "region_weight": 1,
        "region_score": 264178,
        "region_size": 264178,
        "witness_count": 0,
        "slow_score": 0,
        "start_ts": "2023-01-31T03:19:05Z",
        "last_heartbeat_ts": "2023-03-27T06:22:29.244798978Z",
        "uptime": "1323h3m24.244798978s"
      }
    },
  ]
}
```
### What version of PD are you using (`pd-server -V`)?
6.5.0

### PR DESCRIPTION
**Pull request by HuSharp**
### What problem does this PR solve?

Issue Number: Close #6252

### What is changed and how does it work?

```commit-message
ignore the store if the capacity is 0B
```

### Check List

Tests

- Unit test

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by bufferflies — APPROVED** [review]

File: pkg/statistics/store_collection_test.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func TestStoreStatistics(t *testing.T) {
 	re.Equal(0, stats.Unhealthy)
 	re.Equal(0, stats.Disconnect)
 	re.Equal(1, stats.Tombstone)
-	re.Equal(8, stats.LowSpace)
```
Comment (rleungx) [inline pkg/statistics/store_collection_test.go:75]: Maybe use another test to cover the low space situation?

Comment (HuSharp) [reply pkg/statistics/store_collection_test.go:75]: Changed the capacity of one of the stores, now we can know that the `lowspace` is a specified store.

**Review by rleungx — APPROVED** [review]

**Comment by HuSharp** [comment]
It seems that `TestScheduler` has encountered some errors.

File: tests/pdctl/helper.go [hunk 2]
```diff
@@ -35,7 +36,7 @@ import (
 // ExecuteCommand is used for test purpose.
 func ExecuteCommand(root *cobra.Command, args ...string) (output []byte, err error) {
 	buf := new(bytes.Buffer)
-	root.SetOutput(buf)
```
Comment (HuSharp) [inline tests/pdctl/helper.go:38]: update it because linter show `Deprecated: Use SetOut and/or SetErr instead`

**Comment by HuSharp** [comment]
@rleungx @bufferflies have added some prepend settings in tests to pass ci, PTAL, thx!

**Pull request merged**